### PR TITLE
Feature/scone v5

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

**Review by zguesmi — APPROVED** [review]
:+1:

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -57,8 +57,8 @@ scone:
 
 # TODO /!\ remove the option of env variable for releases.
 pre-compute:
-  image: ${IEXEC_TEE_WORKER_PRE_COMPUTE_IMAGE:iexechub/tee-worker-pre-compute:0.1.0-SNAPSHOT-6fc5253} # should accessible in testing pool
-  fingerprint: ${IEXEC_TEE_WORKER_PRE_COMPUTE_FINGERPRINT:39272d644e356773db69e780087be0a820284bc74e139c02ba70683818aa16a0|10b3682afdc726f284403ec78195a590|13076027fc67accba753a3ed2edf03227dfd013b450d68833a5589ec44132100}
+  image: ${IEXEC_TEE_WORKER_PRE_COMPUTE_IMAGE:docker.io/iexechub/tee-worker-pre-compute:7d6bf7e-dev}
+  fingerprint: ${IEXEC_TEE_WORKER_PRE_COMPUTE_FINGERPRINT:0330099545478c4199cc0da6ff7513775e1244bc5c44b1de431df3be43263ff7}
```
Comment (zguesmi) [inline src/main/resources/application.yml:61]: I think we need to add the heap size here with the fingerprint, so it is available for the worker.

Comment (zguesmi) [inline src/main/resources/application.yml:61]: I think we need to add the heap size here with the fingerprint, so it is available for the worker.

**Review by zguesmi — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (jeremyjams) [reply src/main/resources/application.yml:61]: Thank you, will be made in next PRs

**Review by zguesmi — APPROVED** [review]

**Pull request merged**